datamol/io.py

```diff
@@ -95,19 +96,20 @@ def read_sdf(
             `as_df` is True.
         include_computed: Include computed properties in the columns.  Only relevant if
             `as_df` is True.
+        sanitize: Whether to sanitize the molecules
     """
 
     # File-like object
     if isinstance(urlpath, io.IOBase):
-        supplier = Chem.ForwardSDMolSupplier(urlpath)
```

Comment:
@Ishan-Kumar2, the formatting is failing, please run `black` on the files. 

Also for the sanitization, it's better to use the sanitization function  in `datamol` (`sanitize_mol` ),  after loading the SDF,  while keeping the sanitize keyword of the supplier to `False`.

Reply:
@Ishan-Kumar2 could you also propagate `strictParsing`?